content/strings.yml

```diff
@@ -0,0 +1,48 @@
+"Babel could not be loaded": ""
+"This can be caused by an ad blocker. If you're using one, consider adding reactjs.org to the whitelist so the live code examples will work.": ""
+"Loading code example...": ""
+"When you encounter an error, you'll receive a link to this page for that specific error and we'll show you the full error text.": ""
+"The full text of the error you just encountered is:": ""
+"Edit this page": ""
+"We couldn't find what you were looking for.": ""
+"Page Not Found": ""
+"Please contact the owner of the site that linked you to the original URL and let them know their link is broken.": ""
+"Get Started": ""
+"Take the Tutorial": ""
+"A JavaScript library for building user interfaces": ""
+"The React documentation is available in the following languages:": ""
+"In Progress": ""
+"Needs Contributors": ""
+"Don't see your language above?": ""
+"Let us know": ""
+"Contribute": ""
+"Redirecting to Codepen...": ""
+"Not automatically redirecting?": ""
+"All posts ...": ""
+"Recent Posts": ""
+"Previous article": ""
+"Next article": ""
+"React - Acknowledgements": ""
+"We'd like to thank all of our contributors:": ""
+"In addition, we're grateful to": ""
+"<0>Jeff Barczewski</0> for allowing us to use the <1>react</1> package name on npm.": ""
```

Comment:
Could we turn this section into markdown? That way we can just rely on the markdown parser and not have roll out our own markup.

Reply:
That would be great. This way, we can move Acknowledgements and Versions into content. However, I am not sure how they will work because currently there is content and YAML file for both of these pages. What if we use MDX and render the component in the files? Would that be a thing that you will be interested in. So, the resulting MDX will look like this:

```mdx
---
title: Acknowledgement
---

We'd like to thank all of our contributors:

<ContributorsList />

* [Jeff Barczewski](https://github.com/jeffbski") for allowing us to use the [react](https://www.npmjs.com/package/react) package name on npm.
...
```

However, I am not sure how this is going to work if we create a Gatsby theme. Is it possible to inject a component into Markdown Remark per page? If yes, this would actually work (for example, only `ContributorList` will be accessible from Acknowledgements page).

Reply:
@kentcdodds was working on this here: https://github.com/reactjs/reactjs.org/pull/2198

Kent, do you think you could shed some light on Gasim's concerns?

I think MDX would be cool but you're right and I don't know how custom components would work. I think my preference would be:

* If a component is only used in one page, it's okay if it stays in the /content folder next to the page that uses it
* If it's used in multiple pages, put it in custom-components of the Gatsby theme.

Reply:
I feel like there are two routes you can go with enhancing the capabilities of markdown (to make things interactive for example):

1. Make a bunch of custom plugins
2. Use MDX

I think that using MDX is fantastic and reduces more complexity than it adds.

Reply:
Can someone try installing `gatsby-plugin-mdx`. I have been trying to make it work but it keeps failing (Node: v8.16.1):

```bash
yarn add gatsby-plugin-mdx @mdx-js/mdx @mdx-js/react
```

In gatsby-config.js I added `gatsby-plugin-mdx`

Running `yarn start` throws the following errors (I will write the full error message later):

> `createTypes` is undefined
> Cannot call `filter` on undefined